src/index.js

```diff
@@ -1,17 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import Calculator from './components/Calculator';
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
-    <App />
+    <Calculator />
   </React.StrictMode>,
```

Comment:
- [ ] You should try to keep index JS clean, so it should render only `App.js`, and App js file should be the one that imports components like  `Calculator.js`  and renders them. Kindly make sure that the `App.js`(main component) should has the imported `Calculator.js`. App js file should be class based too 😉

Reply:
OK, Thanks for the review, I will resolve that now.